atlas/core/imageManager.hpp

```diff
@@ -15,6 +15,7 @@
 
 namespace imageManager
 {
+	const uint16_t webp_max(16383);
```

Comment:
- `constexpr` should be prefered over `const` in every case where it is possible.
- This file gets included in a lot of places. Why is it in the header and not the place it's directly used? 
- Inline?
- `{}` vs `()` init